src/www/firewall_rules_edit.php

```diff
@@ -1142,7 +1139,7 @@ function is_posnumericint($arg) {
                       <div class="hidden" for="help_for_log">
                         <strong><?=gettext("Log packets that are handled by this rule");?></strong>
                         <br />
-                        <?=gettext("Hint: the firewall has limited local log space. Don't turn on logging for everything. If you want to do a lot of logging, consider using a remote syslog server"); ?> (<?=gettext("see the"); ?> <a href="diag_logs_settings.php"><?=gettext("Diagnostics: System logs: Settings"); ?></a> <?=gettext("page"); ?>).
+                        <?=sprintf(gettext("Hint: the firewall has limited local log space. Don't turn on logging for everything. If you want to do a lot of logging, consider using a remote syslog server (see the %sDiagnostics: System logs: Settings%s page)."),'<a href="diag_logs_settings.php">','</a>') ?>
```

Comment:
would rather try to avoid naming the settings page explicitly as it may change, link directly using "remote syslog server"